src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -498,7 +498,7 @@ function mightCauseHorizontallyOverlappingBubbles(element) {
     //we can be very conservative and say that if the text
     //box isn't taking up the whole width, it *might* cause
     //an overlap
-    if(element.hasClass('bloom-alwaysShowBubble')) {
+    if((element.hasClass) && element.hasClass('bloom-alwaysShowBubble')) {
```

Comment:
If it's generally possible for hasClass to be undefined or not a function, there are 26 other occurrences in this file and goodness knows how many elsewhere. I suppose it has to do with whether the element argument is really a DOM element, and we just need to be careful that we don't ever use it on things that might not be?